### DISCUSSION TITLE
How showing error_log()

### DISCUSSION THREAD

**Discussion by jigujigu** [filamentphp/filament#3795] [Help] [answered]
Hi All,

I trying using error_log('helo bug..'), But I can't find the error message in terminal.
The program is running well, form function running well also.
usually I use this error_log  in my other php project.

```
public static function form(Form $form): Form
{
    error_log('helo bug..');
    return $form
        ->schema([

            Card::make()
                ->schema([
                    TextInput::make('country_code')
                    ->maxLength(3),
                    TextInput::make('name')
                    ->maxLength(255),

                ]),
        ]);
}
```
Thnak you

**Comment by ryangjchandler** [comment, marked as answer]
This function isn't part of Filament and is likely something that exists in another project of yours.
